### ISSUE TITLE
Autocomplete account type

### ISSUE THREAD

**Issue by argaen** [maffin-io/maffin-app#851]
When creating a new account, it's a bit annoying that you have to select the account type after selecting the parent for the cases of expenses/income as there is only one type. We should auto select

**Comment by akhil0203** [comment]
hey @argaen 

I found Maffin to be very interesting project , would love to contribute to this project.
The UI is quite attractive.

**Comment by argaen** [comment]
Hey @akhil0203! glad that you like it :). Do you want to give it a try to this issue? happy to help with the setup or any issues you find

**Comment by akhil0203** [comment]
Yeah i would love to contribute ,but i may take some time have a overall idea about the project thank you for your support. @argaen

**Comment by argaen** [comment]
take your time :). Anything feel free to reach out in discord (in README) or wherever you want!

**Comment by akhil0203** [comment]
> take your time :). Anything feel free to reach out in discord (in README) or wherever you want!

hi @argaen I tried joining discord server but there is some issue with the provided server link in readme.
could you please provide a invite link
![Screenshot (17)](https://github.com/maffin-io/maffin-app/assets/101108396/146bbd80-0125-4f87-be20-bbfdb25930e6)

**Comment by argaen** [comment]
Oops, sry I think I pasted the wrong link, can you try again?

**Comment by akhil0203** [comment]
yeah its working now thanks
